### ISSUE TITLE
Error during Robolectric tests

### ISSUE THREAD

**Issue by guillaume-tgl** [helpscout/beacon-android-sdk-sample#59]
* Beacon version: 1.0.3

I'm using the Beacon SDK in my app and I'm also running tests with Robolectric. Since I updated to 1.0.3, all of my tests fail with the following error:

```
java.lang.IllegalStateException: KoinApplication has not been started

	at org.koin.core.context.GlobalContext.get(GlobalContext.kt:37)
	at org.koin.core.KoinComponent$DefaultImpls.getKoin(KoinComponent.kt:33)
	at com.helpscout.beacon.a.d.a$a.b()
	at com.helpscout.beacon.internal.push.BeaconPushReplyReceiver.getKoin()
	at com.helpscout.beacon.internal.push.BeaconPushReplyReceiver.<init>()
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:423)
	at org.robolectric.util.ReflectionHelpers.callConstructor(ReflectionHelpers.java:379)
	at org.robolectric.internal.bytecode.ShadowImpl.newInstanceOf(ShadowImpl.java:24)
	at org.robolectric.shadow.api.Shadow.newInstanceOf(Shadow.java:35)
	at org.robolectric.android.internal.ParallelUniverse.registerBroadcastReceivers(ParallelUniverse.java:439)
	at org.robolectric.android.internal.ParallelUniverse.setUpApplicationState(ParallelUniverse.java:235)
	at org.robolectric.RobolectricTestRunner.beforeTest(RobolectricTestRunner.java:379)
	at org.robolectric.internal.SandboxTestRunner$2.evaluate(SandboxTestRunner.java:252)
	at org.robolectric.internal.SandboxTestRunner.runChild(SandboxTestRunner.java:130)
	at org.robolectric.internal.SandboxTestRunner.runChild(SandboxTestRunner.java:42)
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:290)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:71)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:288)
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:58)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:268)
	at org.robolectric.internal.SandboxTestRunner$1.evaluate(SandboxTestRunner.java:84)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:363)
	at org.junit.runners.Suite.runChild(Suite.java:128)
	at org.junit.runners.Suite.runChild(Suite.java:27)
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:290)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:71)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:288)
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:58)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:268)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:363)
	at org.junit.runner.JUnitCore.run(JUnitCore.java:137)
	at com.intellij.junit4.JUnit4IdeaTestRunner.startRunnerWithArgs(JUnit4IdeaTestRunner.java:68)
	at com.intellij.rt.execution.junit.IdeaTestRunner$Repeater.startRunnerWithArgs(IdeaTestRunner.java:47)
	at com.intellij.rt.execution.junit.JUnitStarter.prepareStreamsAndStart(JUnitStarter.java:242)
	at com.intellij.rt.execution.junit.JUnitStarter.main(JUnitStarter.java:70)
```

I've found [this issue](https://github.com/InsertKoinIO/koin/issues/503) that seems to be related but I think the fix should happen in your SDK rather than in my app since `Koin` is not exposed outside of the SDK.

**Comment by scottyab** [comment]
Sorry to hear your Robolectric tests are failing due to an issue related to Koin library we use in Beacon SDK. Thanks for the link to the issue. Seems like we need to refactor of our `BeaconPushReplyReceiver` to have Helper object that has the Koin injections in the helper. I'll create an issue for this and get back to you once it's been scheduled to work on.

**Comment by jotaemepereira** [comment]
@guillaume-tgl We just released 1.0.5 with a fix for this issue, let us know how it goes!
